src/detail_screen/anime.js

```diff
@@ -0,0 +1,144 @@
+    author: 'Miura, Kentarou ',
+    genres: ['Action', 'Fantasy', 'Horror'],
+    published: new Date(Date.now()).toLocaleDateString(),
+    rank: '1',
+    mean_score: 8.54,
+    total_votes: 124582,
+    personal_score: '8',
+    status: 'Finished',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Pharetra vel turpis nunc eget lorem dolor sed viverra. Posuere ac ut consequat semper viverra nam libero justo. Nisi vitae suscipit tellus mauris a diam. Enim sit amet venenatis urna cursus eget nunc scelerisque. Sollicitudin ac orci phasellus egestas tellus rutrum tellus. Id semper risus in hendrerit gravida. Enim nec dui nunc mattis enim ut tellus. Rhoncus aenean vel elit scelerisque mauris pellentesque. Bibendum enim facilisis gravida neque convallis. Pretium fusce id velit ut. Arcu non sodales neque sodales ut. Euismod in pellentesque massa placerat. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Pharetra vel turpis nunc eget lorem dolor sed viverra. Posuere ac ut consequat semper viverra nam libero justo. Nisi vitae suscipit tellus mauris a diam. Enim sit amet venenatis urna cursus eget nunc scelerisque. Sollicitudin ac orci phasellus egestas tellus rutrum tellus. Id semper risus in hendrerit gravida. Enim nec dui nunc mattis enim ut tellus. Rhoncus aenean vel elit scelerisque mauris pellentesque. Bibendum enim facilisis gravida neque convallis. Pretium fusce id velit ut. Arcu non sodales neque sodales ut. Euismod in pellentesque massa placerat.',
+    video_url: 'https://www.youtube.com/watch?v=ocQ6PDiP014'
+  },
+
+  shownDialog: false
+}
+
+const reducer = (state, action) => {
+  if (action === 'ANIME/SHOW_DIALOG') {
+    return { ...state, shownDialog: true }
+
+  } else if (action === 'ANIME/HIDE_DIALOG') {
+    return { ...state, shownDialog: false }
+  }
+
+  return state
+}
+
+const Anime = props => {
+  const [state, dispatch] = React.useReducer(reducer, initialState)
+
+  return <>
+    <Header />
+
+    <main>
+      <section className='paper'>
+        <div id='title'>
+          <h1>{ state.anime.name }</h1>
+
+          <IconButton icon='star_rate' onClick={ () => console.log('CLICK') } />
+        </div>
+
+        <img id='main-image' src={ state.anime.image_url } alt='' />
+
+        <p>Episodes: { state.anime.episodes }</p>
+        <p>Author: { state.anime.author }</p>
+        <p>Genres: { state.anime.genres.join(', ') }</p>
+        <p>Published: { state.anime.published }</p>
+      </section>
+
+      <section>
+        <div className='paper'>
+          <h2>Rank #{ state.anime.rank }</h2>
+        </div>
+
+        <div className='paper' id='rating'>
+          <div>
+            <h2>Score</h2>
+            <p>{ state.anime.mean_score }<span>{ state.anime.total_votes } votes</span></p>
+          </div>
+
+          <div className='divider'></div>
```

Comment:
Essa tag pode ser colapsada já que não tem conteúdo dentro
```suggestion
          <div className='divider'/>
```